includes/polling/os/avaya-vsp.inc.php

```diff
@@ -2,15 +2,29 @@
 /*
  * LibreNMS
  *
- * Copyright (c) 2016 Neil Lathwood <neil@lathwood.co.uk>
```

Comment:
You can't remove my copyright, it's fine to add your copyright with it as well but you'll need to restore this line.